Nero/src/NeroLeptons.cpp

```diff
@@ -13,7 +13,7 @@ NeroLeptons::NeroLeptons():
 
     mMinPt_mu = 10;
     mMinEta_mu = 2.4;
-    mMaxIso_mu = 0.2;
+    mMaxIso_mu = 0.4;
```

Comment:
this value is actually overwritten here (keep the -1 that is less strict)
https://github.com/MiT-HEP/NeroProducer/blob/master/Nero/python/Nero_cfi.py#L155